cmd/skr/README.md

```diff
@@ -1,9 +1,10 @@
-The ```skr``` tool instantiates a web server ( http://localhost:<port>) which exposes a REST API so that other containers can retrieve raw attestation via the `attest/raw` POST method and MAA token via the `attest/maa` POST methods as well as release secrets from Azure Key Vault service via the `key/release` POST method. 
+The ```skr``` tool instantiates a web server ( <http://localhost>:<port>) which exposes a REST API so that other containers can retrieve raw attestation via the `attest/raw` POST method and MAA token via the `attest/maa` POST methods as well as release secrets from Azure Key Vault service via the `key/release` POST method.
 
 The tool can be executed using the script [skr.sh](https://github.com/Microsoft/confidential-sidecar-containers/blob/master/skr.sh). If the port number is not specified, the port will default to 8080.

```

Comment:
are we still using `skr.sh`? It looks like this is a dead link to the script

Reply:
maybe more description about the script could be added? like what its inputs are?